src/file_organizer/services/search/bm25_index.py

```
@@ -61,13 +69,11 @@ def index(self, documents: list[str], paths: list[Path]) -> None:
                 f"documents ({len(documents)}) and paths ({len(paths)}) must have equal length"
             )
 
-        try:
-            from rank_bm25 import BM25Okapi  # pyre-ignore[21]
-        except ImportError as exc:
+        if not _BM25_AVAILABLE:
             raise ImportError(
                 "rank-bm25 is required for BM25Index. "
                 "Install it with: pip install 'file-organizer[search]'"
-            ) from exc
+            )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)